packages/smarthr-ui/src/intl/useIntl.ts

```diff
@@ -59,10 +242,58 @@ export const useIntl = () => {
   )
 
   const formatDate = useCallback(
-    (date: Date, opts?: Intl.DateTimeFormatOptions & { jaFormat?: boolean }): string =>
-      intl.formatDate(date, { ...DATE_FORMATS[locale], ...opts }),
+    (props: FormatDateProps): string => {
+      const { date, parts, options } = props
```

Comment:
```suggestion
    ({ date, parts = ['year', 'month', 'day', 'weekday'], options}: FormatDateProps): string => {
```

にすればrequestedPartsを消せて良い気がします！

Reply:
propsは受け取り時に展開してしまいたいです！